DeltGeometriFelleskomponent.TopologyImplementation/GeometryEdit.cs

```diff
@@ -82,8 +84,54 @@ public static TopologyResponse EditLine(EditLineRequest request)
             };
         }
 
+        private static void SetEditOperation(EditLineRequest request)
```

Comment:
Hva gjør vi hvis det trengs med enn en edit for å komme fra gammel til ny geometri?

Reply:
I prinsippet så har vi jo allerede den nye geometrien, så stusser litt på at vi i det hele tatt konstruerer den på nytt.

Reply:
Kaster en NotImplementedException for nå.